### ISSUE TITLE
Result of Keyboard

### ISSUE THREAD

**Issue by muraliq24** [chris-evans/WpfKeyboard#1]
Hi Chris,

I was wondering if it is possible to get the keyboard to automatically populate the value into a text box. 

Thanks,
Murali

**Comment by chris-evans** [comment]
Hi muraliq, I am happy to accept a PR that would do this.  It has been a while since I have touched this, but I believe there is a .com button that would be a good example here. The keyboard send WM_KEYBOARD messages to the OS so the text is going to be input into whatever text box has focus.

**Comment by quicoli** [comment]
> Hi muraliq, I am happy to accept a PR that would do this. It has been a while since I have touched this, but I believe there is a .com button that would be a good example here. The keyboard send WM_KEYBOARD messages to the OS so the text is going to be input into whatever text box has focus.

You are correct on this. I've been actively using this fabulous implementation of  OSK. 
@muraliq24  in my apps here the keyboard is kept in a panel, not visible. So I have a behavior that is called whenever a Textbox receives focus, this triggers an event that brings the keyboard up....
